lib/feature/home/home_page.dart

```diff
@@ -69,6 +70,12 @@ class _LargeHomeView extends StatelessWidget {
                 child: state.selected != null
                     ? DetailView(character: state.selected!)
                     : const Center(child: Text('Select character'))),
+            FloatingActionButton(
+              onPressed: () {
+                GoRouter.of(context).go('/addCharacter');
```

Comment:
navigation should be the same like here https://github.com/arrrtem22/character_viewer/pull/12/files#diff-7d0f01b792ad3bfd3412eb49c3903abf3231bede771b500d6d32f84b37c89c12R106

go_router also uses code generation so do not forget to run make gen after creating new route class

Reply:
Ok
Done